image_view/src/nodes/video_recorder.cpp

```diff
@@ -115,15 +164,20 @@ int main(int argc, char** argv)
     local_nh.param("max_depth_range", max_depth_range, 0.0);
     local_nh.param("use_dynamic_depth_range", use_dynamic_range, false);
     local_nh.param("colormap", colormap, -1);
-
-    if (stamped_filename) {
-      std::size_t found = filename.find_last_of("/\\");
-      std::string path = filename.substr(0, found + 1);
-      std::string basename = filename.substr(found + 1);
-      std::stringstream ss;
-      ss << ros::Time::now().toNSec() << basename;
-      filename = path + ss.str();
-      ROS_INFO("Video recording to %s", filename.c_str());
+    local_nh.param("rolling_buffer_mode", rolling_buffer, false); // rolling_buffer mode enabled/disabled
+    double v_duration;
+    local_nh.param("video_length", v_duration, 5.0);
```

Comment:
Can you please prefix the parameters that are only used for rolling_buffer_mode with `rb_` or `rbf_` to clarify that they aren't generic (e.g. `video_length` could be interpreted as "length of video to record" for any use of this node).

Reply:
Done at 690da7f :+1: